int16   Command
float32 Motor_pos
float32 Motor_vel
float32 Motor_tor
---
bool success   # indicate successful run of triggered service
string message # informational, e.g. for error messages